server/util/redact/redact.go

```diff
@@ -348,6 +348,13 @@ func redactStructuredCommandLine(commandLine *clpb.CommandLine, allowedEnvVars [
 						}
 					}
 				}
+
+				// Redact bazel sub command (for remote runners)
+				if option.OptionName == "bazel_sub_command" {
+					cmdLineTokens := strings.Split(option.OptionValue, " ")
```

Comment:
nit: use `google/shlex` pkg to parse shell tokens - I think this impl won't properly handle stuff like `--remote_header='redact me'`